src/components/FilterFields/__snapshots__/dropDownComponents.test.js.snap

```diff
@@ -5,16 +5,16 @@ exports[`dropDownComponents <AggregationLevel/> matches the snapshot 1`] = `
   label="Aggregation Level"
```

Comment:
Similar feedback as the other snapshot. It's hard to judge as a reviewer whether this snapshot test is still ok (I assume it is since you committed it, but for someone unfamiliar with the app it's hard to judge).

Reply:
Interesting. I agree the snapshot is fairly lengthy but to me personally, when I see the changes in this snapshot it becomes very clear that the `options` prop has changed from an object to an array.

Reply:
https://github.com/dhis2/usage-analytics-app/pull/371#discussion_r486400385

Reply:
Can't improve much on these, and there are other tests in place as well.